x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -873,6 +907,10 @@ class PackagePolicyService implements PackagePolicyServiceInterface {
           namespace: newPolicy.namespace ?? 'default',
           description: newPolicy.description ?? '',
           enabled: newPolicy.enabled ?? true,
+          package: {
+            ...newPP.package!,
+            experimental_data_stream_features: newPolicy.package?.experimental_data_stream_features,
```

Comment:
Should we persist the experimental features on the package policy, or populate that from the package saved object? it seems to me that that feature is shared by all the package policy using this package

Reply:
We only have the value on the package policy here so we can pass it easily to and from the policy editor. It gets persisted on the `epm-package` saved object.

Putting it on `packagePolicy.package` here just made it easiest to access when rendering the policy editor. Open to other approaches.

Reply:
> Putting it on packagePolicy.package here just made it easiest to access when rendering the policy editor. Open to other approaches.

let's consider you have two package policy using nginx package (package policy 1, package policy 2) 
1. you edit package policy 1 to enable the experiment
2. you visit package policy 2 the experiment is shown as disabled but in reality it's enabled no? 

I think we should fetch the installation in the package policy editor.
 
it is clearer?

Reply:
I added logic to `packagePolicyService.get` in this PR to provide the experimental info on `packagePolicy.package` in the API response for package policies. We do fetch from the installation saved object and hydrate the resulting API response body with the experimental feature data. 

It might make a little more sense to return it separately somehow, or to have a separate API request for the policy editor to fetch the installation on its own, though. I could see an argument for that.

Reply:
If we fetch it in the API response there is probably no need to persist on the package policy saved object no?

Reply:
Yeah we don't want to persist it on the package policy saved object. It shouldn't be saved there at all. We are `delete`ing the property before it gets saved. If it's being saved then it's a bug in this PR.